Sources/PackageRegistry/RegistryConfiguration.swift

```diff
@@ -151,5 +350,79 @@ extension RegistryConfiguration: Codable {
         }
 
         try container.encode(self.registryAuthentication, forKey: .authentication)
+        try container.encodeIfPresent(self.security, forKey: .security)
+    }
+}
+
+extension RegistryConfiguration.Security: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case `default`
+        case registryOverrides
+        case scopeOverrides
+        case packageOverrides
+    }
+
+    public init(from decoder: Decoder) throws {
```

Comment:
curios why we need the manual decoding?

Reply:
It thinks `packageOverrides` is an array instead of a dictionary IIRC.